tests/test_units/test_dataio.py

```diff
@@ -121,13 +121,34 @@ def test_deprecated_keys(globalconfig1, regsurf, key, value, wtype, expected_msg
 
 def test_content_not_given(globalconfig1, regsurf):
     """When content is not explicitly given, warning shall be issued."""
-    with pytest.warns(match="The <content> is not provided"):
+    with pytest.warns(UserWarning, match="The <content> is not provided"):
         eobj = ExportData(config=globalconfig1)
         mymeta = eobj.generate_metadata(regsurf)
 
     assert mymeta["data"]["content"] == "unset"
 
 
+def test_content_not_given_case(globalconfig1, regsurf):
```

Comment:
We can revert to this at a later stage, but it's somewhat confusing with the term "case" as it will sometimes mean that we are creating _case_ metadata (class: case) and sometimes that we are exporting _data_ in a _case context_. Could possibly be confusing for new readers at some point...

Reply:
Good point; have fixed this now